docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Conversion/Hash.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class Hash
+    {
+        public static void Example()
```

Comment:
Do we need to at least execute this function in a test?

Reply:
There is a workitem on me to convert all those samples to baseline tests. Can't wait to get to it, for fear of breaking the output through other changes. 

---
In reply to: [267876000](https://github.com/dotnet/machinelearning/pull/3042#discussion_r267876000) [](ancestors = 267876000)

Reply:
[https://github.com/dotnet/machinelearning/issues/2954](https://github.com/dotnet/machinelearning/issues/2954)

---
In reply to: [267958858](https://github.com/dotnet/machinelearning/pull/3042#discussion_r267958858) [](ancestors = 267958858,267876000)